### PR TITLE
WIP: custom splash page for pangeo.pydata.org

### PR DESCRIPTION
**Pull request by jhamman**
I've put together a few modifications that should alter the splash page for pangeo.pydata.org. This is not working just yet but I suspect I'm close enough that @jacobtomlinson can point me in the right direction. 

xref: #78 and https://github.com/pangeo-data/pangeo-custom-jupyterhub-templates

cc @jacobtomlinson and @mrocklin

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
I wouldn't worry about disabling the prepuller. I only did this in ours because I wanted to do it a custom way. My way has actually made it's way upstream now so I could probably remove this too.

I'm surprised it's not working, your config looks the same as mine.

**Comment by jhamman** [comment]
This is working now. We need to update the CSS color scheme and I need to fix a typo in the github issues link but otherwise this should be good to go. This is live now and looks like: 
pangeo_splash

**Comment by kmpaul** [comment]
@jhamman This looks great!  Nice!

**Comment by rabernat** [comment]
You rock Joe!

**Comment by jhamman** [comment]
To be clear, I stole basically everything here from @jacobtomlinson and Jade.

**Comment by mrocklin** [comment]
Woot

**Comment by jacobtomlinson** [comment]
To be clear I stole [it](https://colorlib.com/etc/lf/Login_v1/index.html) from [here](https://colorlib.com/wp/html5-and-css3-login-forms/) and made it orange.

But always happy to take the praise 😄.

File: gce/jupyter-config.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ rbac:
 
 hub:
   extraConfig: |
+    c.JupyterHub.log_level = 'DEBUG'
```
Comment (mrocklin) [inline gce/jupyter-config.yaml]: Do we want to revert this before merging?

**Comment by mrocklin** [comment]
Is there anything left to do here or can this be merged?

**Pull request merged**